plutus-ledger-api/src/PlutusLedgerApi/V2/ParamName.hs

```diff
@@ -200,11 +200,5 @@ data ParamName =
   | ByteStringToInteger'cpu'arguments'c2
   | ByteStringToInteger'memory'arguments'intercept
   | ByteStringToInteger'memory'arguments'slope
-  | LengthOfArray'cpu'arguments
```

Comment:
I can't comment in V3/ParamName.hs, but not that we do need these parameters there, and probably at least the entries for `ListToArray` will need to be changed once we konw what the costing function really is.